ebs-nvme-mapping.sh

```diff
@@ -1,10 +1,21 @@
 #!/bin/bash
```

Comment:
The name of this file doesn't match the udev rule reference.

Reply:
Good catch! Just submitted a fix.